### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1][WIP] List item: Incorrect UI Type read.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11275]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Table with Filer:
The list items within the 'filter' dialog are read out as ‘Link’.. Type of the UI should be read out correctly to screen reader users.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with Filter on toolbar. click filter icon .to open there respective dialog popup. The list items within the 'filter' dialog will be read out as ‘Link’

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,

I tried the filter dialog, but I do not get a link announcement.
Could you please add a screenshot for the exact UI where you observe this?

Thanks,
Nikolay

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov  @droshev  Attached screen reader screenshot
<img width="903" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/69364007-de50-4282-a899-bbcdc6c5bbb6">

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,

I am not able to reproduce this on the provided page: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Maybe in the app, you are using another version.

https://github.com/SAP/fundamental-ngx/assets/17043004/a97fadaf-80b3-4a17-92a5-3e27d1671116

@droshev - By principle there are links there, but they are at the moment not focusable and that is why I assume that the issue is not reproducible. Still, I do not think you need those links. Also the list containing the items needs to be labelled. If you decide to improve this, you can look into the filter dialog in UI5 as a reference: https://ui5.sap.com/#/entity/sap.m.Table/sample/sap.m.sample.TableViewSettingsDialog

Best Regards,
Nikolay

**Comment by JyotiKumari1507** [comment]
@droshev @nikolay-kolarov  I have asked Accessibility team to share recording from their side. I will update once i received it

**Comment by droshev** [comment]
Most of the fields are not filled.

**Comment by JyotiKumari1507** [comment]
@droshev please let us know what fields are pending to fill from our side

**Comment by droshev** [comment]
@JyotiKumari1507 did you set all aria attributes needed? If so, can you create a small stackblitz example demonstrating that you cannot set this on the application side? If not we will close the issue

**Comment by droshev** [comment]
@JyotiKumari1507 We didn't get updates for some time so we are closing the issue. If you think there is a bug, please, create a stackblitz example, document the steps and what the expectation is in a new ticket.

**Comment by divyamalhotra488** [comment]
Created new defect https://github.com/SAP/fundamental-ngx/issues/11964
